templates/.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,15 @@
+name: Pull request
```

Comment:
Mind adding some comments at the top of all the workflows to describe what they are for?